test-configs.yaml

```diff
@@ -756,6 +756,49 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
+  qemu_arm-vexpress-a9:
+    base_name: qemu
+    mach: qemu
+    arch: arm
+    context:
+      arch: 'arm'
+      machine: 'vexpress-a9'
+      cpu: 'cortex-a9'
+      model: 'model=lan9118'
+    dtb: 'vexpress-v2p-ca9.dtb'
+    boot_method: qemu
+    filters:
+      - whitelist: {defconfig: ['vexpress_defconfig']}
+
+  qemu_arm-vexpress-a15:
+    base_name: qemu
+    mach: qemu
+    arch: arm
+    context:
+      arch: 'arm'
+      machine: 'vexpress-a15'
+      cpu: 'cortex-a15'
+      model: 'model=lan9118'
+    dtb: 'vexpress-v2p-ca15-tc1.dtb'
```

Comment:
Only problem here is that when building `vexpress_defconfig`, it didn't create `vexpress-v2p-ca15-tc1.dtb`.  That dtb is there with the `multi_v7_defconfig` build though, but then it's filtered out.

So the staging run only included `versatile-pb` which has its dtb created with `vexpress_defconfig`:
https://staging.kernelci.org/boot/id/5d8be5fecf3a0fe289322dc6/

Reply:
arch/arm/configs/vexpress_defconfig contains CONFIG_ARCH_VEXPRESS=y
And in arch/arm/boot/dts/Makefile, I see that vexpress-v2p-ca15-tc1 is built with ARCH_VEXPRESS.
So it is strange that vexpress_defconfig didnt create it.

Reply:
I have just compiled a vexpress_defconfig and express-v2p-ca15-tc1.dtb is built

Reply:
Fair enough, tried this with mainline v5.4-rc8 and it seems to do the job:
```
$ make dtbs
scripts/kconfig/conf  --syncconfig Kconfig
  UPD     include/config/kernel.release
  HOSTCC  scripts/dtc/dtc.o
[...]
  HOSTLD  scripts/dtc/dtc
  DTC     arch/arm/boot/dts/vexpress-v2p-ca5s.dtb
  DTC     arch/arm/boot/dts/vexpress-v2p-ca9.dtb
  DTC     arch/arm/boot/dts/vexpress-v2p-ca15-tc1.dtb
  DTC     arch/arm/boot/dts/vexpress-v2p-ca15_a7.dtb
```

Reply:
I'll enable `vexpress_defconfig` in today's staging run to check it works all the way and gets tested on QEMU as expected.

Reply:
OK got a vexpress_defconfig build on staging: https://staging-storage.kernelci.org/kernelci/staging.kernelci.org/staging-20191125.0/arm/vexpress_defconfig/gcc-8/

It was tested in several labs so that seems fine:
https://staging-ci.kernelci.org/job/test-runner/751/console

However there's still no boot results here:
https://staging.kernelci.org/boot/all/job/kernelci/branch/staging.kernelci.org/kernel/staging-20191125.0/

I'm investigating this now.  We should also run the baseline test plan on these QEMU variants.

Reply:
I think the problem was because of this: https://github.com/kernelci/kernelci-backend/pull/180

Testing again on staging now with the backend fix for Unicode characters.

Reply:
Finally got a boot result now: https://staging.kernelci.org/boot/id/5ddd3eff092e747463ed498e/

So this is fine - except the baseline test plan should also be run, but I can add that.